src/Process.php

```diff
@@ -35,6 +35,8 @@ class Process extends EventEmitter
     private $stopSignal;
     private $termSignal;
 
+    private $windowsWorkaround = false;
```

Comment:
As per the discussion we should consider if we might want to default this to `true` on Windows?